experiments/ClimaEarth/components/ocean/oceananigans.jl

```diff
@@ -95,17 +82,19 @@ function OceananigansSimulation(area_fraction, start_date, stop_date; output_dir
     # Set initial condition to EN4 state estimate at start_date
     OC.set!(ocean.model, T = en4_temperature[1], S = en4_salinity[1])
 
+    # Construct a remapper from the exchange grid to `Center, Center` fields
     long_cc = OC.λnodes(grid, OC.Center(), OC.Center(), OC.Center())
     lat_cc = OC.φnodes(grid, OC.Center(), OC.Center(), OC.Center())
 
-    # TODO: Go from 0 to Nx+1, Ny+1 (for halos) (for LatLongGrid)
-
     # Construct a remapper from the exchange grid to `Center, Center` fields
     long_cc = reshape(long_cc, length(long_cc), 1)
     lat_cc = reshape(lat_cc, 1, length(lat_cc))
     target_points_cc = @. CC.Geometry.LatLongPoint(lat_cc, long_cc)
```

Comment:
Looks like the memory error caused by huge allocation is hit here. What is the command doing?

Reply:
This creates a Matrix of size length(lat_cc) x length(long_cc) containing a (lat, long) pair at each point

Reply:
this is probably the issue, `lat_cc` and `long_cc` are 2D matrices for a tripolar grid. We can use those ones directly